.github/workflows/jdl.yml

```diff
@@ -42,24 +42,30 @@ jobs:
     runs-on: ${{ matrix.os }}
     defaults:
       run:
+        shell: bash
         working-directory: ${{ github.workspace }}/app
     if: github.event.action != 'closed'
     timeout-minutes: 90
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-11]
+        os: [ubuntu-20.04, macos-11, windows-2022]
```

Comment:
When I tried to use windows-latest instead of windows-2019, I ran out of disk space. https://github.com/graalvm/setup-graalvm/issues/6#issuecomment-1058328963